server/package.json

```diff
@@ -16,7 +16,9 @@
         "aws-xray-sdk-core": "^3.1.0",
         "body-parser": "^1.19.0",
         "bufferutil": "^4.0.1",
+        "connect-dynamodb": "^2.0.4",
```

Comment:
is this dependency needed? Can we not just use the AWS SDK?

Reply:
I can look into it, it pairs nicely with express-session. Can you add 'research how to use amazon sdk' to issues for this goal?